container-chains/nodes/frontier/src/rpc/mod.rs

```diff
@@ -355,7 +356,7 @@ where
 {
     // Frontier offchain DB task. Essential.
     // Maps emulated ethereum data to substrate native data.
-    match params.frontier_backend {
+    match &*params.frontier_backend {
```

Comment:
Probably to allow pattern matching on an `Arc`, see: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=9e10ecf1d77ee0c47b30012e24f77982

(that example works with just `*` because `u8` implements `Copy`, otherwise you need `&*`)

Reply:
I see, need to learn more about Arcs. Thanks a lot!